.npmrc

```diff
@@ -0,0 +1 @@
+tag-version-prefix=""
```

Comment:
By default, NPM prefixes Git tags with `v` (e.g. `v1.0.0`). This setting changes the prefix to an empty string